Pricing and Quotas content update (#1808)
getsentry/sentry-docs: src/collections/_documentation/accounts/quotas/index.md

sidebar_order: 0
---

> Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events - errors, attachments, and transactions - to be tracked. Each of these type of events has a quota. When an event is accepted by Sentry, it counts toward your quota for that type of event.

Author: Sure. Thanks.

Sentry’s flexibility means you can exercise fine-grained control over which events count toward your quota.

## Key Terms

Let’s clarify a few terms to start:

-   Event - an event is one instance of you sending Sentry data. Generally, this data is an error. Every event has a set of characteristics, called its fingerprint.

> -   Issue - an issue is a grouping of similar events, which all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).

Reviewer: ```suggestion
-   Issue - an issue is a grouping of similar events, which all share the same fingerprint. For example, Sentry groups events together the same part of your code triggers them. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
```

Author: I'm not sure that edit clarifies.

Reviewer: Ah, you're right. I think I meant, "For example, Sentry groups events together when the same part of your code triggers them." But, I think the original version also works. Was trying to work it into active voice.

Author: In some cases, we gotta go with the passive voice :)

> -   Attachment - Attachments are files uploaded in the same request, such as log files. Unless the option to store crash reports is enabled, Sentry will use the files only to create the event, then will drop the files. 

Reviewer: ```suggestion
-   Attachment - Attachments are files uploaded in the same request, such as log files. Unless the option to store crash reports is enabled, Sentry will use the files only to create the event, and then drop the files. 
```

Reviewer: The last sentence took me a couple reads so I tried rephrasing.

Author: Thanks :)

> -   Transaction - A transaction represents a single instance of a service being called to support an operation your want to measure or track, like a page load.

Reviewer: ```suggestion
-   Transaction - A transaction represents a single instance of a service being called to support an operation you want to measure or track, like a page load.
```

Author: Thanks for catching the typo. I'm going to keep the ordering as is since folks need to read the preceding definitions to understand the final definition (quota).

5. **Spike protection**

>       Sentry’s spike protection prevents huge overages from consuming your event capacity; spike protection is not current available for transactions. For more information, see [Spike Protection](/accounts/quotas/#spike-protection).

Reviewer: ```suggestion
      Sentry’s spike protection prevents huge overages from consuming your event capacity; spike protection is not currently available for transactions. For more information, see [Spike Protection](/accounts/quotas/#spike-protection).
```

Author: Thanks for the second set of eyes.

In addition, depending on your project’s configuration and the plan you subscribe to, Sentry may also check:

6. **Rate limit for the project**

>       If the errror event rate limit for the project has been exceeded, and your subscription allows, the event will not be counted. For more information, see [Rate Limiting in our guide to Manage Your Event Stream](/accounts/quotas/manage-event-stream-guide/#4-rate-limiting) or  [Rate Limiting Projects](/accounts/quotas/#id1).

Reviewer: ```suggestion
      If the error event rate limit for the project has been exceeded, and your subscription allows, the event will not be counted. For more information, see [Rate Limiting in our guide to Manage Your Event Stream](/accounts/quotas/manage-event-stream-guide/#4-rate-limiting) or  [Rate Limiting Projects](/accounts/quotas/#id1).
```

Author: Thanks.
